kzz7yh-f58197/cod-ve09v2_kzz7yh-f58197.xml

```diff
@@ -356,8 +361,8 @@
       <lb ed="#V" n="107"/>in ea fuisset sacta aliqua claustrorum apertio: ergo necvaegitatis amissio. 
    </p>
                <p xml:id="kzz7yh-f58197-d1e647">
-                  <lb ed="#V" n="108"/>Contra vrginem concipere et pere miraculum est: sed in statu 
-      <lb ed="#V" n="109"/>inocentie mulier concepisset: et peperisset naturaliter: ergo 
+                  <lb ed="#V" n="108"/>Contra virginem concipere et perere miraculum est: sed in statu 
```

Comment:
here the per/par abbreviation needs to be interpreted because sometimes it can stand for "par" as is the case here.

"parere" meaning "to bear" or "give birth to"

```suggestion
                  <lb ed="#V" n="108"/>Contra virginem concipere et parere miraculum est: sed in statu 
```